src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Rendering/EventForDotNet.ts

```diff
@@ -69,18 +74,137 @@
       case 'pointerout':
       case 'pointerover':
       case 'pointerup':
-        return new EventForDotNet<UIPointerEventArgs>('pointer', { Type: event.type });
+        return new EventForDotNet<UIPointerEventArgs>('pointer', parsePointerEvent(event));
 
+      case 'wheel':
       case 'mousewheel':
-        return new EventForDotNet<UIWheelEventArgs>('wheel', { Type: event.type });
-
+        return new EventForDotNet<UIWheelEventArgs>('wheel', parseWheelEvent(event));
 
       default:
-        return new EventForDotNet<UIEventArgs>('unknown', { Type: event.type });
+        return new EventForDotNet<UIEventArgs>('unknown', { type: event.type });
     }
   }
 }
 
+function parseDragEvent(event: any) {
+  return {
```

Comment:
Unimportant stylistic point: with TypeScript this can be expressed without duplicating the key names, e.g.:

    return { event.type, event.detail, event.dataTransfer, ...}

It compiles to the same result.

Reply:
ahhh ok ;) live and learn 👍 

Fixing.

Reply:
Looks like it didn't like that:
![image](https://user-images.githubusercontent.com/4714040/39947052-eae21196-5546-11e8-8549-7a7ea6a5bf92.png)